test_1.js

```diff
@@ -0,0 +1,687 @@
+/**
+ * Copyright 2018 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const path = require('path');
+const puppeteer = require('puppeteer-core');const findChrome = require('./find_chrome');
+const {rpc} = require('../rpc');
+const debugApp = require('debug')('carlo:app');
+const debugServer = require('debug')('carlo:server');
```

Comment:
format: style mismatch:
```suggestion
const debugServer = require('debug') ('carlo:server');
```

␣ should be inserted at column 37.